Customer-research/User-research-file.md

```diff
@@ -1,3 +1,16 @@
 # Reseach on applicant attitudes
 
 ## 🤔 Research question: Why are so many applicants dropping before uploading resumes or portfolios? 
+
+### Required years of experience seem to high to some applicants
+
+> "A lot of these job postings are from big companies, like FAANG or adjacent. I think I might have a better chance at a smaller, lesser known company and I don't know what those companies might be. I also struggle with junior or associate positions that state you need 3 or 5 years experience. How is that even junior!?!?"
+
+-jacquelinhiggins
+
+
+### Portfolios are a lot of work and don't represent my experience
+
+> "The biggest pain points I've found include building a portfolio that's succinct, complete and accurately reflects who I am and what I do. Another pain point with the portfolio is including the right kind and amount of information."
```

Comment:
This is so true, we saw this in our interviews last week as well - I'll look up the quote and add it